File: Parth Mathur/81_maximumScore.java
class Solution {
    public int maximumScore(int a, int b, int c) {
     int[] nums=new int[]{a,b,c};
     Arrays.sort(nums);
     if(nums[2]>nums[1]+nums[0]){
         int sum=nums[0]+nums[1];
         return sum;
     }
        else   {
               int sum=a+b+c;
              return sum/2;
        }
    }
}

File: Dhairyashil-Shinde/81. Maximum Score.cpp
class Solution {
public:
    int maximumScore(int a, int b, int c) {
        a = min(a, b + c);
        b = min(b, a + c);
        c = min(c, a + b);
        return (a + b + c) >> 1;
    }
};
